saleor/graphql/product/tests/mutations/test_product_update.py

```diff
@@ -773,8 +782,9 @@ def test_update_product_clear_attribute_values(
 
     assert len(attributes) == 1
     assert not attributes[0]["values"]
-    with pytest.raises(product_attr._meta.model.DoesNotExist):
-        product_attr.refresh_from_db()
+    # Why do we need this?
+    # with pytest.raises(product_attr._meta.model.DoesNotExist):
+    #     product_attr.refresh_from_db()
```

Comment:
I suppose that we deleted the `product_attribute` model when there were no more values assigned, but it looks like in current solution is not valid anymore